frame/balances/src/tests.rs

```diff
@@ -28,15 +28,15 @@ macro_rules! decl_tests {
 		use frame_support::{
 			assert_noop, assert_storage_noop, assert_ok, assert_err,
 			traits::{
-				LockableCurrency, LockIdentifier, WithdrawReasons,
+				fungible, fungible::Lockable, WithdrawReasons,
```

Comment:
What would happen if you don't include `fungible::Lockable`?

Reply:
I get this, if fungible::Lockable isn't included:
![image](https://user-images.githubusercontent.com/6564482/207858510-a0b06e8b-9d6f-4e2e-a0e9-c4863a5bf28d.png)

Reply:
ouch, got it :)